.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Run Benchmark
+
+on:
+  push:
+    branches: [master]
```

Comment:
In wonder, should this run on dev branches, too ? If I touch the algorithm this early feedback could be useful.